frontend/src/bundles/talent-onboarding/helpers/map-experience-years-to-marks/map-experience-years-to-marks.ts

```diff
@@ -31,6 +31,13 @@ const experienceYearsSliderMarks = experienceYearsScaled.map((mark) => {
     };
 });
 
+const formatNumber = (number: number): number => {
+    const INT_TO_FIXED = 1;
+    const parsedNumber = Number.parseFloat('' + number);
+
+    return Number.parseFloat(parsedNumber.toFixed(INT_TO_FIXED));
+};
```

Comment:
I think we can move it to helpers

Reply:
Hello, this is based off of an earlier version of #344. This has been addressed there in a newer version.